### PR TITLE
Atlases & CAT12

### LINKED ISSUE #56: Duplicated lines in AICHA ROI file
TL;DR: `G_Temporal_Pole_Mid-2-L` region is duplicated in [`lut_AICHA_newformat.txt`](https://github.com/aramis-lab/clinica/blob/dev/clinica/resources/atlases_spm/lut_AICHA_newformat.txt) file

See [Google Groups discussion for details](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/clinica-user/kOGQ0laeMfc/jmOgXJFpBAAJ).

Verification on other atlases seems to be necessary.

### PR DESCRIPTION
**Pull request by alexandreroutier**
- Remove CAT12 from dependencies
- Simplify & harmonize how atlases are handled
- Resolves #56
- Add checksum for FSL atlases

### CODE REVIEW

**Comment by alexandreroutier** [comment]
Hi @mdiazmel,

This PR is ready for ready for review :)

One last commit will be done but this involves `T1_Volume.md` and  `PET_Volume.md` so the code can be reviewed.

Best,
Alex

File: clinica/utils/inputs.py [hunk 1]
```diff
@@ -394,3 +394,24 @@ def fetch_file(remote, dirname=None):
...
+    if not(os.path.exists(cache_clinica)):
+        os.makedirs(cache_clinica)
+
+    local_file = os.path.join(cache_clinica, remote_file.filename)
+
+    if not(os.path.exists(local_file)):
+        try:
+            local_file = fetch_file(remote_file, cache_clinica)
+        except IOError as err:
+            cprint('Unable to download %s from %s: %s' % (remote_file.filename, remote_file.url, err))
```
Comment (mdiazmel) [inline clinica/utils/inputs.py:415]: I'm in campaign to switch to the new _f-string_ format everywhere ;)! Here, for example, something like this should work:
```suggestion
            cprint(f"Unable to download {remote_file.filename} from {remote_file.url}: {err}")
```

**Review by mdiazmel — COMMENTED** [review]
Hi, @alexandreroutier ,
I suppose that the package of the next release will be much lighter (maybe some hundreds of Kb) thanks to you ;)!
I added a small comment about using _f-string_ everywhere is possible. Take a look and then I will proceed to merge.

**Review by mdiazmel — COMMENTED** [review]
I'll merge this PR!

**Pull request merged**